src/malli/util.cljc

```diff
@@ -373,6 +373,21 @@
                               (apply f (get s k) args))))]
     (up schema ks f args)))
 
+(defn extend-multi
```

Comment:
what is the difference with `extend-multi` and `assoc`?

```clojure
(mu/extend-multi [:multi {:dispatch :type} [:a :string] [:b :string]] [:b :int])
; => [:multi {:dispatch :type} [:a :string] [:b :int]]

(mu/assoc [:multi {:dispatch :type} [:a :string] [:b :string]] :b :int)
; => [:multi {:dispatch :type} [:a :string] [:b :int]]
```

Reply:
```clojure
(extend-multi [:multi {:dispatch :type} [:a :string] [:b :string]] [:c :int])
; => [:multi {:dispatch :type} [:a :string] [:b :string] [:c :int]]

(mu/assoc [:multi {:dispatch :type} [:a :string] [:b :string]] :c :int)
; => [:multi {:dispatch :type} [:a :string] [:b :string] [:c :int]]
```

Reply:
Ah, I think they're the same. I initially thought the `::m/default` case would need special handling but then rolled that back.